.github/workflows/versioned-coverage.yml

```diff
@@ -1,38 +0,0 @@
-# This workflow is intended to be used to run versioned tests for different scenarios(i.e.- legacy context manager, etc)
-
-name: Nightly Versioned Scenario Runs
-
-on:
-  workflow_dispatch:
-  schedule:
-    - cron:  '0 9 * * 1-5'
-
-env:
-  # Enable versioned runner quiet mode to make CI output easier to read:
-  OUTPUT_MODE: quiet
-
-jobs:
-  legacy-context:
-    runs-on: ubuntu-latest
-
-    strategy:
-      fail-fast: false
-      matrix:
-        node-version: [18.x, 20.x, 22.x]
-
-    steps:
-    - uses: actions/checkout@v4
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v4
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Install Dependencies
-      run: npm install
-    - name: Run Docker Services
-      run: npm run services
-    - name: Run Legacy Context Versioned Tests
```

Comment:
For those reading this later: this is the key bit, it was only running legacy context manager tests.